### PR TITLE
Form changed done

### PR DESCRIPTION
**Pull request by TeroOtavia**
Form changed:
- Pedagogisen tuen perusteet select field replaved with Tuen tarve textarea

Resolves: #6902

### CODE REVIEW

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/pedagogy-PDF.tsx [hunk 1]
```diff
@@ -289,38 +280,12 @@ const PedagogyPDF = (props: PedagogyPDFProps) => {
         </View>
 
         <View style={styles.infoFieldContainer}>
-          <Text style={styles.infoFieldLabel}>
-            {t("labels.basisForSupport", {
-              ns: "pedagogySupportPlan",
-              context: "pedagogy",
-            })}
+          <Text style={styles.infoFieldLabel}>Tuen tarve</Text>
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/pedagogy-PDF.tsx:283]: Lokalisointi

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/steps/need-of-support-information.tsx [hunk 2]
```diff
@@ -152,71 +127,18 @@ const NeedOfSupportInformation: React.FC<NeedOfSupportInformationProps> = (
...
-                context: "pedagogy",
-              })}
-              noOptionsMessage={() =>
-                t("content.empty", {
-                  ns: "pedagogySupportPlan",
-                  context: "options",
-                })
+            <Textarea
+              id="needOfSupport"
+              label="Tuen tarve"
```
Comment (avukkonen) [inline muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/steps/need-of-support-information.tsx:132]: Lokalisointi

**Review by avukkonen — CHANGES_REQUESTED** [review]

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/steps/need-of-support-information.tsx (hunk 2 above)
Comment (jjlankinen) [reply muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/steps/need-of-support-information.tsx:132]: Korjattu.

File: muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/pedagogy-PDF.tsx (hunk 1 above)
Comment (jjlankinen) [reply muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/general/pedagogical-support-form/pedagogy-PDF.tsx:283]: Korjattu.

**Review by avukkonen — APPROVED** [review]

**Pull request merged**